src/popup/pages/Names/AuctionHistory.vue

```diff
@@ -107,8 +108,8 @@ export default defineComponent({
       justify-content: space-between;
     }
 
-    .account-item {
-      margin-top: 4px;
+    .account-item :deep(.name) {
```

Comment:
Since `AccountItem` component already have `size` prop and it is only font related and not length (width) i suggest to use `:deep` here for now. Since the AENS is goin to be redesign.

Reply:
I strongly advise against using `:deep` - you know that :)